thirdeye/thirdeye-frontend/app/pods/components/te-modal/component.js

```diff
@@ -64,16 +65,16 @@ export default Component.extend({
      * Performs the cancel action and closes the modal
      */
     async onCancel() {
-      await this.send('cancelAction');
```

Comment:
Blocking comment: This is breaking the implementation of the modal component. Please pass the action into te-modal as 
```
{{#te-modal 
  saveAction=(action "onSave")
  cancelAction=(action "onCancel")
}}
```

Reply:
fixed